src/Directory.Build.targets

```diff
@@ -64,12 +67,13 @@
 	<PropertyGroup>
 		<!-- IMPORTANT!!! KEEP BOTH MSTest and MTP versions in sync -->
 		<!-- For a given MSTest version, go to https://www.nuget.org/packages/MSTest.TestAdapter/3.7.1#dependencies-body-tab -->
-		<!-- (replacing 3.7.1 in the url with the MSTest version in question) -->
+		<!-- (Or, for dogfooding versions, go to https://dnceng.visualstudio.com/public/_artifacts/feed/test-tools/NuGet/MSTest.TestAdapter/overview/3.8.0-preview.25067.5) -->
+		<!-- (replacing the version in the url with the MSTest version in question) -->
 		<!-- And look for MTP version in there, and make sure MicrosoftTestingPlatformVersion is the same -->
 		<!-- Alternatively, migrate to MSTest.Sdk and let it handle the MTP versioning for you -->
 		<!-- In case of MSTest.Sdk, the TrxReport package will not be needed. It will be added automatically -->
-		<MSTestVersion>3.7.1</MSTestVersion>
-		<MicrosoftTestingPlatformVersion>1.5.1</MicrosoftTestingPlatformVersion>
+		<MSTestVersion>3.8.0-preview.25067.5</MSTestVersion>
+		<MicrosoftTestingPlatformVersion>1.6.0-preview.25067.5</MicrosoftTestingPlatformVersion>
```

Comment:
We can setup a canary for that one, once it builds fully, and stay on the stable for the main builds.

Reply:
Sure, let me know once you set it up please.

Meanwhile, I will revert the version change, and I think it should be good to proceed with the rest of the changes?

Reply:
Hmm, part of the changes here are using APIs that are new in 3.8, so this can't get merged with using 3.7.1.

Reply:
3.8.0 stable is out, so this PR is unblocked now.

@jeromelaban Let me know please if you will setup a canary.

Reply:
We're modifying the tree signficantly for now, we'll setup a canary after we're done with it. We'll still be able to merge this PR once it's green.

Reply:
Thank you!
Hopefully the current run should be green.